commands/core/docs.drush.inc

```diff
@@ -65,6 +65,22 @@ function docs_drush_command() {
     'callback' => 'drush_print_file',
     'callback arguments' => array($docs_dir . '/examples/example.drushrc.php'),
   );
+  $items['docs-filter-config'] = array(
```

Comment:
lets do docs-config-filter and docs-config-exporting. that will sort the commands closer and will help with disambiguation versus drupal filter system.